### DISCUSSION TITLE
Custom domain is not working

### DISCUSSION THREAD

**Discussion by TidharHayute** [vercel/community#4712] [Help] [answered]
### Summary

Hey there, 
I just deployed a Git repository to Vercel's hosting.
The website loads under the 'free' domain that comes with the project ([showcard-io.vercel.com](showcard-io.vercel.com)), but then I added a custom domain (showcard.io), The DNS configures are working, as you can see here:

<img width="550" alt="Screenshot 2023-11-02 at 20 28 14" src="https://github.com/vercel/community/assets/139790265/3d6fa571-8984-4538-a10c-d5b2a854abf5">

Yet, when I open the custom domain link ([showcard.io](showcard.io)) I get this error:

<img width="550" alt="Screenshot 2023-11-02 at 20 29 32" src="https://github.com/vercel/community/assets/139790265/b4c51010-eaab-4868-a95f-8ec5758edf3f">

### Steps to Reproduce

The error I received:

<img width="550" alt="Screenshot 2023-11-02 at 20 29 32" src="https://github.com/vercel/community/assets/139790265/b4c51010-eaab-4868-a95f-8ec5758edf3f">

Is there someone here who can help me fix this issue, or at least explain to me why it happens?

**Comment by amyegan** [comment, marked as answer]
Hey @TidharHayute! It looks like you're using Cloudflare based on the [NS record for your domain](https://www.whatsmydns.net/#NS/showcard.io). We have some documentation to help you get the domain working correctly with Cloudflare and your Vercel Project. Please let me know if you still have trouble after using the settings recommended there 🙏 

https://vercel.com/docs/integrations/cloudflare

**Reply by TidharHayute** [reply]
Hey Amy, thank you for your answer. I followed the documentation but the website is still not loading with the custom domain. I tried both methods but they didn't work..

**Reply by acidio** [reply]
Thanks @amyegan , I was having the same issue and the article you shared helped to solve it. In my case, as I'm using as a reserve proxy, it was missing to enable the Full encrypt option, after doing it, It worked.

Exactly step: https://vercel.com/docs/integrations/cloudflare#set-cloudflare-ssl/tls-encryption-mode
